opentok/opentok.py

```diff
@@ -1599,6 +1600,164 @@ def set_broadcast_layout(self, broadcast_id, layout_type, stylesheet=None, scree
         else:
             raise RequestError("OpenTok server error.", response.status_code)
 
+    def start_render(self, session_id, opentok_token, url, max_duration=7200, resolution="1280x720", status_callback_url=None, properties: dict = None):
+        """
+        This method starts a render for the specified session.
+
+        :param String 'session_id': The ID of a session (generated with the same `APIKEY` as specified in the URL) which you wish to start rendering into.
+        :param String 'token': A valid OpenTok token with a Publisher role and (optionally) connection data to be associated with the output stream.
+        :param String 'url': A publically reachable URL controlled by the customer and capable of generating the content to be rendered without user intervention.
+        :param Integer 'maxDuration' Optional: The maximum time allowed for the Render, in seconds. After this time, the Render will be stopped automatically, if it is still running.
+        :param String 'resolution' Optional: Resolution of the display area for the composition.
+        :param String 'statusCallbackUrl' Optional: URL of the customer service where the callbacks will be received. 
```

Comment:
We are not documenting this REST API option. Developers should use account portal to register callbacks. See https://tokbox.com/developer/guides/experience-composer/#configuring-callbacks. (Please delete throughout.)

Reply:
Have deleted this variable throughout